.github/workflows/parameters_check.yml

```diff
@@ -37,7 +36,15 @@ jobs:
 
       - name: Perform security check
         run: |
-          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 0 "boolean_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 1 "boolean_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 2 "boolean_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 0 "shortint_classic_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 1 "shortint_classic_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 2 "shortint_classic_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 0 "shortint_multi_bit_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 1 "shortint_multi_bit_parameters_lattice_estimator"
+          PYTHONPATH=lattice_estimator sage ci/lattice_estimator.sage 2 "shortint_multi_bit_parameters_lattice_estimator"
```

Comment:
this is likely unnecessary by auto partitioning in the sage script ?

also if this is an issue we can just change the runs-on of the job to e.g. large_ubuntu_16 see line 17, this way the change to the sage script may not be required anymore

Reply:
The problem is that releasing memory from a single run of the script seems to be a bit problematic -- the CI machine runs out of RAM and crashes. Splitting up the task in this way with some command line args is a bit of an easy way around it